docs/control-structures.rst

```diff
@@ -228,6 +230,9 @@ With the pragma on, internal functions remain unlocked by default but can still
 .. note::
    The ``nonreentrancy on/off`` pragma is scoped to the current file. If you import a file without the ``nonreentrancy on`` pragma, the functions in that file will behave as the author intended, that is, they will be reentrant unless marked otherwise.
 
+.. note::
+    The ``constant`` and ``imutable`` state variable getters don't check the lock because the value of the variables can't change.
```

Comment:
```suggestion
    The ``constant`` and ``immutable`` state variable getters don't check the lock because the value of the variables can't change.
```